docs/source_analysis.md

```diff
@@ -1,33 +1,30 @@
 # Source Analysis
```

Comment:
Okay, overall these are good descriptions of each of these sites. But, you can say anything about their utility.... are there any biases detectable? Who funds these institutions? How might their perspective shape how they present the data?